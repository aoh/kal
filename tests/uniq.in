# flags: -t 0
1.1.1970
 - foo
 - bar
 - foo

every Thursday: foo
