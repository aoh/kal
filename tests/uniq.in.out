1.1.1970, Thursday, week 1
 - bar
 - foo
